cerberus_full_package/CERBERUS_V2/panel_v2/public/bot.sql

```diff
@@ -51,7 +51,7 @@ CREATE TABLE `bots` (
   `activeDevice` varchar(254) DEFAULT NULL,
   `timeWorking` varchar(254) DEFAULT NULL,
   `statDownloadModule` varchar(3) DEFAULT NULL,
-  `statAdmin` varchar(3) DEFAULT NULL,
```

Comment:
false

Reply:
falsee

Reply:
yurivolkov this man is a scammer I have all the evidence.